### PR TITLE
Removed useless TTT util functions

### PR DESCRIPTION
**Pull request by Kefta**
I never knew UnbridledHateForULX existed, made me laugh a bit :V:

### CODE REVIEW

File: garrysmod/gamemodes/terrortown/gamemode/lang_shd.lua [hunk 1]
```diff
@@ -6,7 +6,7 @@
 if LANG then return end
 LANG = {}
 
-util.IncludeClientFile("cl_lang.lua")
+if SERVER then AddCSLuaFile("cl_lang.lua") else include("cl_lang.lua") end
```
Comment (Bo98) [inline garrysmod/gamemodes/terrortown/gamemode/lang_shd.lua]: You forgot about line 18.

**Comment by Kefta** [comment]
Is there a way to squash commits from the website? I'm short of a computer that I can use git on for the time being

**Comment by Bo98** [comment]
Not that I'm aware of.

File: garrysmod/gamemodes/terrortown/gamemode/util.lua [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-
 if not util then return end
 
+local util = util
 local math = math
 local string = string
 local table = table
 local pairs = pairs
+local table = table
+local band = bit.band
```
Comment (TheFreeman193) [inline garrysmod/gamemodes/terrortown/gamemode/util.lua]: Couldn't you just modify L305 to use bit.band?

Comment (Kefta) [inline garrysmod/gamemodes/terrortown/gamemode/util.lua]: I'm not sure if it actually has any efficiency benefits, but they're all over TTT, so I won't touch them unless all of them are changed

File: garrysmod/gamemodes/terrortown/gamemode/lang_shd.lua (hunk 1 above)
Comment (Kefta) [inline garrysmod/gamemodes/terrortown/gamemode/lang_shd.lua]: Fixed, thanks

**Comment by Kefta** [comment]
I'm sorry for the amount of commits; I modified my master branch on my fork by accident, so I don't have the ability to squash them

**Comment by svdm** [comment]
You've mixed in a lot of changes that are opinion rather than clear improvements.

* Replacing a bunch of IncludeClientFile calls with oneline if statements, just to remove 7 lines of code? That's replacing code with more code.
* Similarly, your change of GetRTranslation just ends up duplicating the role_strings lookup. What's the gain exactly?
* The cases of a local being set for a module function (e.g., `local exp = math.exp`) save a dictionary lookup when that function is called, which has benefits if that function is called very often. They have a purpose and are not exactly harmful, so why remove them?

Really the only useless function this PR removes is the ULX one.

**Comment by Kefta** [comment]
1 + 2. It's still better than creating a global function to only be used in only file. In passthrough's case, it is only used one time. I could see the benefit of putting IncludeClientFile as a local function in that file since it's not used anywhere else, but passthrough is definitely unnecessary.

3. That's my fault, I thought I fixed but it didn't appear that the commit went through. 

I'll redo the PR if you tell me what to do about IncludeClientFile and passthrough

**Comment by Kefta** [comment]
1+2. But they're only used in one file and you made them global functions. I see the benefit of adding IncludeClientFile as a local function to lang_shd, but passthrough has no use whatsoever.
3. I thought I fixed that; I guess my commit didn't go through. I'll revert those changes once you tell me what to do about IncludeClientFile and passthrough.

**Comment by svdm** [comment]
Global? They're in a module, specifically util, where they are quite harmless. Both of them should be left alone, simply because removing them only achieves making the code more verbose.

util.passthrough is the identity function, ie. it returns its argument. This is a common concept in functional programming languages where you are often passing functions around into other functions. It's useful in the same way that, sometimes, multiplying a number by 1 is useful.

**Pull request closed without merging**